### ISSUE TITLE
Inconsistent inference results between PyTorch and converted TensorRT model using with Normalize operator

### ISSUE THREAD

**Issue by Thrsu** [NVIDIA-AI-IOT/torch2trt#903]
### Description:

I'm experiencing a discrepancy between the inference results of PyTorch model and the TensorRT model obtained by converting it using the torch2trt tool.

### Reproduce

This issue can be reproduced by the following script:
```python
import torch
from torch.nn import Module
from torch2trt import torch2trt

input_data = torch.randn([9, 2], dtype=torch.float32).cuda()
class normalize(Module):
    def forward(self, *args):
        return torch.nn.functional.normalize(args[0], p=85,)
model = normalize().float().eval().cuda()
model_trt = torch2trt(model, [input_data])
output = model(input_data)
output_trt = model_trt(input_data)
print(torch.max(torch.abs(output - output_trt)))
```

### Environment

- torch: 2.1.1
- torch2trt: 0.4.0
- tensorrt: 8.6.1
